database/factories/UserFactory.php

```diff
@@ -20,13 +20,11 @@ public function definition(): array
         $faker = \Faker\Factory::create();
 
         return [
-            'name' => $faker->firstName(),
-            'surname' => $faker->lastName(),
-            'dni' => $faker->unique()->regexify('[A-Z0-9]{9}'),
+            'username' => $faker->userName(),
+            'dni' => $faker->unique()->regexify('[1-9]{8}[A-Z]'),
             'email' => $faker->unique()->safeEmail(),
             'email_verified_at' => now(),
             'password' => bcrypt('password123'), // O puedes usar Faker para generar contraseñas aleatorias
```

Comment:
I'd prefer to leave the factory more versatile allowing to create a random password and then, when I use it, I can pass a customized one by parameter.